src/sentry/incidents/logic.py

```diff
@@ -313,6 +313,8 @@ def bulk_build_incident_query_params(incidents, start=None, end=None):
             "start": incident.date_started if start is None else start,
             "end": incident.current_end_date if end is None else end,
         }
+        # Make start about 20% earlier:
+        params["start"] = params["start"] - (params["end"] - params["start"]) / 5
```

Comment:
Looks good. Did we have a minimum amount of time we wanted to show at the start of the graph? If an incident started a couple of minutes ago we might still want to show some larger amount of data.